index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;600;700&display=swap" rel="stylesheet">
-    <title>Foga Portfolio Website</title>
+    <title>Foga Kater Portfolio Website</title>
 </head>
 
 <body>
```

Comment:
- [ ] Good job working on the accessibility 👍, but still need to to fix some points 😃. Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text

![Capture](https://user-images.githubusercontent.com/57408419/229777163-8857104c-38fc-427f-9bcc-c6abd5c7fbd4.JPG)